e2e/test/scenarios/collections/uploads.cy.spec.js

```diff
@@ -263,13 +263,13 @@ function uploadFile(testFile, valid = true) {
     );
   });
 
-  cy.findByRole("status").within(() => {
+  cy.findAllByRole("status").within(() => {
```

Comment:
I don't think this will work. When it's flake you'll get 2 elements and `within` would fail.

I've tested the similar command with
```ts
      cy.findAllByRole("link").within(() => {});
```

And got this error
```
cy.within() can only be called on a single element. Your subject contained 6 elements. Narrow down your subject to a single element (using .first(), for example) before calling .within().

To run .within() over multiple subjects, use .each().

cy.get('div').each($div => {
  cy.wrap($div).within(() => { ... })
})[Learn more](https://on.cypress.io/within)
```
![image](https://github.com/metabase/metabase/assets/1937582/facea26f-b629-42fa-bffc-e7bebe835a31)

Reply:
Stress-test reveals 20/20 passes for all three affected tests
https://github.com/metabase/metabase/actions/runs/6823741815/job/18558232573#step:9:197

Let me double and triple check things locally.

Reply:
@WiNloSt you're absolutely right. Thank you for catching this.
I've found a workaround and applied it in https://github.com/metabase/metabase/pull/35579/commits/054d24a91b933dfd88c3df85817cd383dad0c165. Mind taking another look? I'll do a few more stress-tests in CI in meantime.